ibis/backends/dask/tests/test_client.py

```diff
@@ -136,3 +135,9 @@ def test_invalid_connection_parameter_types(npartitions):
     )
     with pytest.raises(TypeError, match=expeced_msg):
         ibis.dask.from_dataframe("file.csv")
+
+
+def test_has_operation(client):
```

Comment:
Any reason not to make this a backend test? Seems simple enough to `never` the ones that do support it.

Reply:
Bug felt specific enough to these three backends. Happy to move the test if you think that'd be better.

Reply:
Let's put this into a single backend test.

Reply:
Done